src/QMCHamiltonians/HamiltonianFactory.cpp

```diff
@@ -808,13 +808,13 @@ bool HamiltonianFactory::put(xmlNodePtr cur)
 {
   bool success=build(cur,false);
   
-  if(targetH->EnableVirtualMoves)
-  {
-    app_log() << "  Hamiltonian enables VirtualMoves" << std::endl;
-    targetPtcl->enableVirtualMoves();
-  }
-  else
-    app_log() << "  Hamiltonian disables VirtualMoves" << std::endl;
+  //if(targetH->EnableVirtualMoves)
```

Comment:
I think this was no more used even in the AoS and the printout is no more relevant.